sourcegraph/conn_pool.go

```diff
@@ -0,0 +1,43 @@
+package sourcegraph
+
+import (
+	"sync"
+
+	"google.golang.org/grpc"
+)
+
+var (
+	conns   map[string]*grpc.ClientConn
+	connsMu sync.Mutex // guards conns
```

Comment:
Minor, it's more readable style to write the guard mutex as a "header" and all the vars it guards directly underneath, as in this example:

http://gotools.org/github.com/bradfitz/http2#transport-go-L31-L32

So this can be rewritten as:

```Go
var (
	connsMu sync.Mutex
	conns   map[string]*grpc.ClientConn
)
```

The "// guards conns" comment becomes implicit.